installers/conda/win/project.nsi

```diff
@@ -0,0 +1,232 @@
+    FileOpen $0 "$INSTDIR\bin\MantidWorkbench-script.pyw" w # This w is intentional and opens it in write mode
+    FileWrite $0 "#!$INSTDIR\bin\pythonw.exe$\n"
+    FileWrite $0 "import workbench.app.main$\n"
+    FileWrite $0 "workbench.app.main.main()$\n"
+    FileClose $0
+
+    # Store installation folder in registry
+    WriteRegStr HKCU "Software\${PACKAGE_VENDOR}\${PACKAGE_NAME}" "" $INSTDIR
+
+    # Make an uninstaller
+    WriteUninstaller $INSTDIR\Uninstall.exe
+
+	# Write registry entries for uninstaller for "Add/Remove programs" information
+	WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PACKAGE_NAME}" "DisplayName" "${DISPLAY_NAME}"
+	WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PACKAGE_NAME}" "UninstallString" "$\"$INSTDIR\Uninstall.exe$\""
+	WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PACKAGE_NAME}" "DisplayVersion" "${VERSION}"
+	WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PACKAGE_NAME}" "NoRepair" 1
+	WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PACKAGE_NAME}" "NoModify" 1
+	WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PACKAGE_NAME}" "DisplayIcon" "$\"$INSTDIR\Uninstall.exe$\""
+	WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PACKAGE_NAME}" "Publisher" "${PACKAGE_VENDOR}"
+# See if this is automatically added by the MUI stuff?
+#	WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PACKAGE_NAME}" "StartMenu" "${StartMenuFolder}"
+
+    # Create shortucts for start menu
+    !insertmacro MUI_STARTMENU_WRITE_BEGIN Application
+        CreateDirectory "$SMPROGRAMS\$StartMenuFolder"
+        CreateShortCut "$SMPROGRAMS\$StartMenuFolder\${MANTIDWORKBENCH_LINK_NAME}" "$INSTDIR\bin\MantidWorkbench.exe"
+        SetOutPath "$INSTDIR\bin" # Not sure why this is here?
+        CreateShortCut "$SMPROGRAMS\$StartMenuFolder\${MANTIDNOTEBOOK_LINK_NAME}" "cmd.exe" "/C $\"call $INSTDIR\bin\pythonw.exe -m notebook --notebook-dir=%userprofile%$\"" "${NOTEBOOK_ICON}"
+        SetOutPath "$INSTDIR" # Not sure why this is here?
+        CreateShortCut "$SMPROGRAMS\$StartMenuFolder\Uninstall.lnk" "$\"$INSTDIR\Uninstall.exe$\""
+    !insertmacro MUI_STARTMENU_WRITE_END
+
+    # Create desktop shortcuts
+    CreateShortCut "$DESKTOP\${MANTIDWORKBENCH_LINK_NAME}" "$INSTDIR\bin\MantidWorkbench.exe" "" "${WORKBENCH_ICON}"
+    SetOutPath "$INSTDIR\bin" # Not sure why this is here?
+    CreateShortCut "$DESKTOP\${MANTIDNOTEBOOK_LINK_NAME}" "cmd.exe" "/C $\"call $INSTDIR\bin\pythonw.exe -m notebook --notebook-dir=%userprofile%$\"" "${NOTEBOOK_ICON}"
+    SetOutPath "$INSTDIR" # Not sure why this is here?
+
+SectionEnd
+
+# The uninstall section
+Section "Uninstall"
+    # Remove all files and folders written by the installer
+    # Should we instead get the install location from registry? Here $INSTDIR is wherever the uninstaller .exe is run.
```

Comment:
Again - not sure about this comment.

Reply:
https://github.com/mantidproject/mantid/issues/33811